release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -107,6 +107,7 @@ With this release, there are several updates to the *Developer* perspective of t
 * Export your application in the ZIP file format to another project or cluster by using the *Export application* option on the *+Add* page.
 * Create a Kafka event sink to receive events from a particular source and send them to a Kafka topic.
 * Set the default resource preference in the *User Preferences* -> *Applications* page. In addition, you can select another resource type to be the default.
+** Select another resource type from the list of resource types if required by browsing the Add page and clicking *Import from Git* -> *Advanced options* -> *Resource type* drop-down list.
```

Comment:
Set another resource type from the Add page. Click Import from Git -> Advanced options -> Resource type to set the resource from the drop-down list.

Reply:
@smunje1: My suggested version (basically personal touch of what @DebarghoGhosh has said) is given below. You can safely use what Debargho suggested.

Set another resource type from the **Add** page by clicking **Import from Git** -> **Advanced options** -> **Resource type** and selecting the resource from the drop-down list.

Reply:
@sounix000 and @DebarghoGhosh If you see https://github.com/openshift/openshift-docs/pull/51733, we have covered it as an Optional Step (Please see Step 7), so I believe we should retain "if required" here. I am planning to rephrase it as follows:

Set another resource type from the **Add page** if required by clicking **Import from Git** -> **Advanced options** -> **Resource type** and selecting the resource from the drop-down list.

@sounix000 @DebarghoGhosh WDYT?

Cc @divyanshiGupta @sanketpathak

Reply:
@smunje1 : In that case, you can start the sentence as "Optionally, set the ... ". Because your use of "if required" in the middle of the sentence does not convey any other specific condition, other than implying that this action is optional under some conditions.

Also, in the actual content, "Add page" is not in bold.